src/components/Clock/StopwatchComponent/StopwatchComponent.mjs

```diff
@@ -0,0 +1,108 @@
+import css from "./StopwatchComponent.scss";
+import html from "./StopwatchComponent.html?raw";
+
+export class StopwatchComponent extends HTMLElement {
+    constructor() {
+        super();
+
+        const shadow = this.attachShadow({ mode: "open" });
+        shadow.innerHTML = html;
+
+        // Import style
+        const style = document.createElement("style");
+        style.textContent = css;
+        shadow.appendChild(style);
+
+        this.timerDisplay = shadow.getElementById("stopwatch-timer");
+        this.lapTimer = shadow.getElementById("lap-timer");
+
+        this.lapList = shadow.getElementById("lap-list");
+        this.resetButton = shadow.getElementById("reset-stopwatch");
+        this.startButton = shadow.getElementById("start-stopwatch");
+        this.lapButton = shadow.getElementById("lap-stopwatch");
+
+        this.isStarted = false;
+        this.isPaused = false;
+        this.time = 0;
+        this.timer = null;
+        this.timerLaps = [];
+
+        //listeners
+        this.startButton.addEventListener("click", () => {
+            if (this.isStarted) {
+                this.isPaused ? this.resumeTimer() : this.pauseTimer();
+            } else {
+                this.isStarted = true;
+                this.time = 0;
+                this.timerLaps.push(0);
+                this.startTimer();
+            }
+        });
+
+        this.resetButton.addEventListener("click", () => this.resetTimer());
+
+        this.lapButton.addEventListener("click", () => this.isStarted && this.addLap());
+    }
+
+    addLap() {
+        let newLap = this.time;
```

Comment:
Pourquoi assigner this.time dans une nouvelle variable ?
De plus, cette variable et celles en dessous n'ont pas besoin d'être en let.